File: examples/node/apigateway/directory-routing/api/handler/_router.js
const {Router} = require('@syngenta-digital/alc').apigateway;

exports.route = async (event) => {
    const router = new Router({
        routingMode: 'directory',
        basePath: 'directory-example',
        handlerPath: 'api/handler',
        schemaPath: 'openapi.yml'
    });
    return router.route(event);
};

File: examples/apigateway/directory/api/v1/handler/__router.py
from acai_aws.apigateway.router import Router


router = Router(
    base_path='directory-example/v1',
    handlers='api/v1/handler',
    schema='api/openapi.yml'
)
router.auto_load()

def route(event, context):
    return router.route(event, context)